examples/00-EDB/10_GDS_workflow.py

```diff
@@ -0,0 +1,104 @@
+"""
+EDB: Edit Control File and import gds
+-------------------------------------
+This example shows how you can use PyAEDT to import a gds from an IC file.
+"""
+
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes importing a section.
+
+import os
+import tempfile
+import pyaedt
+import shutil
+from pyaedt.edb_core.edb_data.control_file import ControlFile
+
+###############################################################################
+# Download file
+# ~~~~~~~~~~~~~
+# Download the AEDB file and copy it in the temporary folder.
```

Comment:
Is it an AEDB file or GDS file? Is an ICS file a type of AEDT file? Sorry for my ignorance. I just want to check for clarity and consistency.